packages/babel-traverse/src/path/index.js

```diff
@@ -146,6 +147,10 @@ export default class NodePath {
     if (!debug.enabled) return;
     debug(`${this.getPathLocation()} ${this.type}: ${message}`);
   }
+
+  toString() {
+    return generator(this.node).code;
```

Comment:
Or could we somehow make this change in babel-core which does have it as a dep?

Reply:
any ideas on how that would look like? at the moment it's as easy as doing:
```js
CallExpression(path) {
  console.log(path + '')
}
```